index.html

```diff
@@ -6,6 +6,10 @@
   <title>Rock Paper Scissors</title>
 </head>
 <body>
-  <script src="script.js"></script>
+  <script src="script.js" defer></script>
```

Comment:
Just a note, while using `defer` works, it's also common practice (and prefered in some cases) to just put all the JS at the end of the `<body>`.